### ISSUE TITLE
add works to the top of works for you feed

### ISSUE THREAD

**Issue by katarinabatina** [artsy/eigen#1477]
In preparation for universal link support from personalized emails, a user who clicks on an artist name from the email should see the new works by that artist on the top of their feed.

**Comment by alloy** [comment]
Does it have to be at the top or can we just set the scroll position so that the selected artist is visible?

**Comment by katarinabatina** [comment]
@alloy we could but then I think we'd need to create a "more artworks above" button (similar to facebook or twitter)

**Comment by alloy** [comment]
What about scrolling down to the artist in question on showing the view?

**Comment by alloy** [comment]
(In an animated way, that is, thus showing that there are works above it.)

**Comment by katarinabatina** [comment]
@alloy sounds like we may need to address this issue on Eigen which is somewhat related https://github.com/artsy/force/issues/5015

**Comment by katarinabatina** [comment]
@alloy just followed the rabbit hole of this ticket down the the gravity PR haha...can you summarize what impact the fix in Gravity has on Eigen and if scrolling to the proper entry will be feasible given the fix (if the fix was implemented I'm not sure?)

**Comment by alloy** [comment]
It's not really related to this issue. The linked issue was more a bug that could cause some works never showing up.
